packages/dialtone-vue2/components/popover/popover.vue

```diff
@@ -492,7 +492,7 @@ export default {
     /**
      * Sets the element to which the popover is going to append to.
      * 'body' will append to the nearest body (supports shadow DOM).
```

Comment:
Add a comment explaining what does root means here, something like: "When the popover anchor is inside an iframe,  'root' will attach the popover to the iFrame parent's body"